packages/parser-sass/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "dependencies": {
+    "@hint/utils": "^4.1.0",
+    "postcss": "^7.0.17",
+    "postcss-scss": "^2.0.0",
+    "postcss-sass": "^0.4.1"
+  },
+  "description": "webhint parser needed to analyze SASS and SCSS files",
+  "devDependencies": {
+    "@hint/parser-css": "^3.0.8",
+    "@types/node": "^12.7.3",
+    "@types/proxyquire": "^1.3.28",
+    "@types/sinon": "^7.0.13",
+    "@typescript-eslint/eslint-plugin": "^1.13.0",
+    "@typescript-eslint/parser": "^1.12.0",
+    "ava": "^1.4.1",
+    "cpx": "^1.5.0",
+    "eslint": "^5.15.1",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-markdown": "^1.0.0",
+    "eventemitter2": "^5.0.1",
+    "hint": "^5.2.2",
+    "npm-link-check": "^3.0.0",
+    "npm-run-all": "^4.1.5",
+    "nyc": "^14.1.0",
+    "proxyquire": "2.0.0",
+    "rimraf": "^2.6.3",
+    "sinon": "^7.4.2",
+    "typescript": "^3.6.2"
+  },
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "dist/src"
+  ],
+  "homepage": "https://webhint.io/",
+  "keywords": [
+    "sass",
+    "scss",
+    "webhint",
+    "webhint-parser"
+  ],
+  "license": "Apache-2.0",
+  "main": "dist/src/parser.js",
+  "name": "@hint/parser-sass",
+  "nyc": {
+    "extends": "../../.nycrc"
+  },
+  "peerDependencies": {
+    "hint": "^5.2.2"
+  },
+  "repository": {
```

Comment:
Do we want a `"private": true` here or are we ready to publish it directly?

In the past we were doing it because of the documentation, right? But now that we get everything from npm it should be fine?

Reply:
Yeah, I don't think we need to keep packages `private` anymore unless we're expecting to do a release without them. This one should be ready to go once we merge.